lib/govuk_publishing_components/presenters/component_wrapper_helper.rb

```diff
@@ -0,0 +1,59 @@
+module GovukPublishingComponents
+  module Presenters
+    class ComponentWrapperHelper
+      def initialize(options)
+        @options = options
+
+        if @options.include?(:classes)
+          classes = @options[:classes].split(" ")
+          unless classes.all? { |c| c.start_with?("js-", "gem-c-", "govuk-") }
```

Comment:
Great spot. Yes, it is currently possible to pass an invalid class to a component using this method. Will fix.

Reply:
Have modified this so that classes are checked both when initially passed and when passed using the helper method. I've also implemented similar checking on the `id` and `aria` attributes.